ClangSharp/Generated/CXAvailabilityKind.cs

```diff
@@ -2,9 +2,9 @@ namespace ClangSharp
 {
     public enum CXAvailabilityKind
     {
-        CXAvailability_Available = 0,
-        CXAvailability_Deprecated = 1,
-        CXAvailability_NotAvailable = 2,
-        CXAvailability_NotAccessible = 3,
+        CXAvailability_Available,
```

Comment:
The header file doesn't explicitly set the values for most enums and we don't need to either.